cmd/experimental/kueue-viz/backend/handlers/gvr.go

```diff
@@ -0,0 +1,86 @@
+package handlers
+
+import (
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+// Define the GroupVersionResource for ResourceFlavors
+func ResourceFlavorGVR() schema.GroupVersionResource {
+	return schema.GroupVersionResource{
+		Group:    "kueue.x-k8s.io",
+		Version:  "v1beta1",
+		Resource: "resourceflavors",
+	}
+}
+
+// Define the GroupVersionResource for ClusterQueues
```

Comment:
```suggestion
// ClusterQueueGVR define the GroupVersionResource for ClusterQueues
```

Or remove it.

Reply:
fixed, added all and uniform naming, all in plural